src/test/java/EulerTest.java

```diff
@@ -0,0 +1,62 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EulerTest {
+    @Test
+    void whenUpperLimitIs10_Return23() {
```

Comment:
Zamiast `Return` raczej stosujemy `ThenShouldReturn` ale ciężko coś mówić, bo tak jak mówiłem na zajęciach - to jest wszystko kwestia konwencji przyjętej w projekcie.